### PR TITLE
Add batch operations, confirm destructive actions, ensure a token remains active

### PR DESCRIPTION
**Pull request by gber**

### CODE REVIEW

**Review by dzatoah — COMMENTED** [review]
Testing worked after a few fixes

File: personal/privacyidea/class_PrivacyIdeaUtils.inc [hunk 1]
```diff
@@ -339,7 +339,9 @@ class PrivacyIdeaUtils implements PILog
         },  []);
     }
 
-    /** @return string Token serial */
+    /** @param string Token serial
+     *  @return bool Whether token is valid
```
Comment (dzatoah) [inline personal/privacyidea/class_PrivacyIdeaUtils.inc:343]: s/Whether/Wether

**Review by dzatoah — COMMENTED** [review]
Looks good! Let me test this...

File: personal/privacyidea/class_PrivacyIdeaUtils.inc (hunk 1 above)
Comment (gber) [reply personal/privacyidea/class_PrivacyIdeaUtils.inc:343]: https://www.merriam-webster.com/dictionary/wether ;-)

Comment (dzatoah) [reply personal/privacyidea/class_PrivacyIdeaUtils.inc:343]: Oh hahaha, ignore me then :D

**Pull request merged**